docs/advanced/writing-declarations.md

```diff
@@ -530,6 +532,27 @@ const v3 = (v1 * 4) as Vector;
 const d = v3.dot(v2);
 ```
 
+There are also language extensions that allow declaring special functions which will transpile to operators. For example:
```

Comment:
```suggestion
The second option was added in version [0.38.0](https://github.com/TypeScriptToLua/TypeScriptToLua/blob/master/CHANGELOG.md#0380). You can now use [language extensions](https://typescripttolua.github.io/docs/advanced/language-extensions) that allow declaring special functions which will transpile to operators. This will be completely type safe if the operators are declared correctly.

Example:
```

Maybe mention this one first?
I think this should be the recommended variant.